resources/js/processes-catalogue/components/menuCatologue.vue

```diff
@@ -139,11 +174,20 @@ export default {
       this.select(item);
     },
     selectTemplateItem(item) {
+      if (item.label === "All Templates") {
```

Comment:
This will not work if the application is open in spanish, please use an id is a bad practice use a label

Reply:
Solved. Logic was changed to item.id